src/helm/benchmark/window_services/image_generation/lexica_search_window_service.py

```diff
@@ -0,0 +1,20 @@
+from .clip_window_service import CLIPWindowService
+from helm.benchmark.window_services.tokenizer_service import TokenizerService
+
+
+class LexicaSearchWindowService(CLIPWindowService):
```

Comment:
Does this window service (and other window services in this pull request) have the same `truncate_from_right()` quirk as CLIPWindowService?

Reply:
No, so I overrode the parent method.